### PR TITLE
Only query for givpower of boosters

### PR DESCRIPTION
**Pull request by aminlatifi**

### CODE REVIEW

File: src/repositories/userPowerRepository.test.ts [hunk 1]
```diff
@@ -103,4 +135,36 @@ function findUsersThatDidntSyncTheirPowerTestCases() {
     // usersWithoutPowerSync should not include user1
     assert.isNotOk(usersWithoutPowerSync.find(user => user.id === user1.id));
   });
+  it('should return users without a boosted any project', async () => {
```
Comment (CarlosQ96) [inline src/repositories/userPowerRepository.test.ts]:
```suggestion
  it('should not return users without any boosted project', async () => {
```

Comment (CarlosQ96) [reply src/repositories/userPowerRepository.test.ts]: Just a text suggestion to make it clear, I was getting confused as the headline was not what was ocurring on the code execution.

**Review by CarlosQ96 — APPROVED** [review]
Code is fine! Thanks for the updates. 

Minor suggestion: I would change the description of the test to make it clearer vs what it's doing.

After that you can merge. (also testing suggestions like you told me 😄 )

File: src/repositories/userPowerRepository.test.ts (hunk 1 above)
Comment (aminlatifi) [reply src/repositories/userPowerRepository.test.ts]: Thanks @CarlosQ96

**Pull request merged**